tests/unit/Core/YourMembershipExceptionTest.php

```diff
@@ -22,8 +22,10 @@ protected function _after()
     // tests
     public function testToString()
     {
-        $exception = new YourMembershipException('Message', 999, 'TestMethod');
+        //setup
+        $exception = new YourMembershipApiException('Message', 999, 'TestMethod');
 
+        //verify
         echo $exception;
```

Comment:
It would be good to see an assertion in this test. Can you call toString() on it explicitly and assert the value is what you'd expect?